packages/db/test/pinning.spec.js

```diff
@@ -364,43 +315,64 @@ describe('Pin Request', () => {
       assert.strictEqual(prs.length, 10)
     })
 
+    it('returns the total count', async () => {
+      const { count } = await client.listPsaPinRequests(authKeyPinList)
+      assert.strictEqual(count, totalPinned)
+    })
+
     it('limits the results to the provided limit', async () => {
       const limit = 8
-      const { results: prs } = await client.listPsaPinRequests(authKeyPinList, {
+      const { results: prs, count } = await client.listPsaPinRequests(authKeyPinList, {
         limit
       })
       assert.strictEqual(prs.length, limit)
+      assert.strictEqual(count, totalPinned)
     })
 
     it('returns only requests for the provided token', async () => {
-      const { results: prs } = await client.listPsaPinRequests('10')
+      const { results: prs, count } = await client.listPsaPinRequests('10')
       assert.strictEqual(prs.length, 0)
+      assert.strictEqual(count, 0)
     })
 
     it('sorts by date', async () => {
-      const { results: prs } = await client.listPsaPinRequests(authKeyPinList)
+      const { results: prs, count } = await client.listPsaPinRequests(authKeyPinList)
 
       const sorted = prs.reduce((n, item) => n !== null && item.created <= n.created && item)
       assert(sorted)
+      assert.strictEqual(count, totalPinned)
     })
 
     it('filters items by provided status', async () => {
-      const { results: pins } = await client.listPsaPinRequests(authKeyPinList, {
+      const { results: pins, count } = await client.listPsaPinRequests(authKeyPinList, {
         statuses: ['Pinning', 'PinError']
       })
 
       assert.strictEqual(pins.length, 2)
+      assert.strictEqual(count, 2)
```

Comment:
nit: it would be nice to not hardcode the number and count the actual value from `pinRequestsInputs`, to avoid future test failures with more fixtures added

Reply:
I see your point, at the same time this feels like the usual balance of maintainable tests vs striving to not write tests that are too clever. I feel in this case it is better to hardcode it.